src/Avalonia.Base/Input/Gestures.cs

```diff
@@ -10,8 +10,29 @@ namespace Avalonia.Input
 {
     public static class Gestures
     {
-        private static bool s_isDoubleTapped = false;
-        private static bool s_isHolding;
+        private readonly struct GestureState
```

Comment:
You can use record instead
```suggestion
        private record struct GestureState(GestureStateType Type, IPointer Pointer, Point Point);
```

Reply:
Done.